comid/svn.go

```diff
@@ -217,7 +217,7 @@ func (o TaggedMinSVN) Equal(r TaggedMinSVN) bool {
 	return ret == 0
 }
 
-// Compare helper function to compare two SVNs, object and reference
+// Compare is a helper function to compare two SVNs, object and reference
```

Comment:
It's a [Godoc convention](https://tip.golang.org/doc/comment) to start the comment on a function with the name of the function.

```suggestion
// compare is a helper function to compare two SVNs, object and reference
```